app/models/concerns/with_qualifications.rb

```diff
@@ -16,19 +16,19 @@ module WithQualifications
       # Professional: the student will receive a Professional Graduate
       # Certificate of Education (offered at Level 6) or Professional
       # Graduate Diploma in Education (PGDE), with no credits or
-      # modules at postgraduate (master’s) Level 7 on successful
+      # modules at postgraduate (master's) Level 7 on successful
```

Comment:
btw, I'm not being petty with changing the apostrophe to single-quotes, I did it just because this was forcing my editor to stick a utf-8 coding magic comment at the top of the file and I thought may as well keep these clean ASCII for now. This was probably the wrong decision. ¯\_(ツ)_/¯